packages/database/src/migrations/20231112212117-AddProjectIdToSurveys-modifies-schema.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.runSql(`ALTER TABLE survey ADD project_id TEXT NULL`);
```

Comment:
I'm not sure of the syntax but we should also set this as a foreign key. Something like `FOREIGN KEY (project_id) REFERENCES project(id)` 

Also I checked with the others and we generally add an index for any foreign keys. Again I'm not 100% sure on the syntax.